### ISSUE TITLE
[USER STORY] Read trasactions for credit risk analytics

### ISSUE THREAD

**Issue by data-david** [Informasjonsforvaltning/behov#157]
As a: bank
I wish to: Read trasactions 
So that: I can perform acredit risk analytics

Relatert til epos: #4 

Lenke til design: 
Lenke til løsningsarkitektur:

**Comment by MadsBDamgaard** [comment]
Is the below text way too much info for the current purpose?
User Story 2: Credit Assessment

User Aim: 
A Credit Assessor from Bank X wants to evaluate the risk of offering a company credit. This is done by reading/querying the company about a number of financial figures relating to that company’s production and consumption during the very latest period (e.g. during the last three months), in some cases with the aim of comparing the latest period to the same period in other years, or comparing to financial statements from recent years.


User Story, details (from end user point of view)

Credit Assessor queries API about the aggregated figures for the company’s accounts for a determinate period. The accounts queried are those also found on annual financial statements, so that the assessor can calculate the trends: 
GrossProfit
ProfitLoss from ordinary operating activities
Distributions
Assets (intangible, etc)
Property, plant, equipment,
Inventories
Equity
Cash
Liabilities
provisions
A complete list is defined by the actual testing organization. Suggestions: P. 32 in https://drive.google.com/drive/folders/1qDnu241X-LcBy4aaTpgGKJZaPEM7CArr 

2. API outputs a XBRL file

3. Credit assessor loads the XBRL file in his credit assessment system

**Comment by ekoskent** [comment]
The quering may also be done by a separate micro service rather than the API (query API creation is not prio 1). Some notes about the data available through the invoices bank statements in relation to the data needs mentioned above:
GrossProfit - can be done
ProfitLoss from ordinary operating activities- can be done
Distributions - what does this mean?
Assets (intangible, etc) - this data we don't currently have as in list of assets but accounting entries from the current period we have. Perhaps we should create the opening balances for the balance sheet manually?
Property, plant, equipment, -same applies for all of these..
Inventories
Equity
Cash
Liabilities
provisions

**Comment by MadsBDamgaard** [comment]
There is probably a better word for "Distribution" - something that describes the amount of money taken out by the owner or shareholders of a business. "Dividends" perhaps?

**Comment by data-david** [comment]
Do you agree that we move this story out of the Reference Implementation and into a separate microservice placeholder for the Credit PoC ?  @MadsBDamgaard @ekoskent

**Comment by MadsBDamgaard** [comment]
I am not competent to assess what that means, unfortunately. (so I am not able to say anything clever about choosing not to query via API but via microservice) 
I guess you have a good reason to do so!

**Comment by data-david** [comment]
No difference. It is just a question on what you get out of the first API. I would prefer that the business logic of filtering is external to the bookkeeping. So want to split the PoC in a seperat epic.

**Comment by MadsBDamgaard** [comment]
good! That makes sense to me 👍

**Comment by ekoskent** [comment]
I agree with David to remove it as i see it's not a separate API but a micro service created for this task and using the general read entries API.
